Translation/en/RedirectedResources/assets/abdata/adv/scenario/c28/00/1008/translation.txt

```diff
@@ -1,28 +1,28 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「夢を見ているかと錯覚してしまいそうな光景じゃのう……」=
-//「海ほたるだね。すごい数だ」=
-//「そうか……話には聞いておったが、これが海ほたるなのじゃな」=
-//「チラチラと波間にきらめく、この淡く儚い光……」=
-//「まさしく、海のほたるじゃな」=
-//「そう……だね、大きさもふわふわ飛んでるあの蛍と同じくらいかな？」=
-//「うむ、かもしれぬ」=
-//「この光の粒、ひとつひとつが命なのじゃな……」=
+「夢を見ているかと錯覚してしまいそうな光景じゃのう……」=“What a sight, it is enough to make one think they're in a dream...”
+「海ほたるだね。すごい数だ」=“Oh, it's the sea fireflies. So many of them too.”
+「そうか……話には聞いておったが、これが海ほたるなのじゃな」=“I see... I've heard of them before, so these are those sea fireflies, huh?”
+「チラチラと波間にきらめく、この淡く儚い光……」=“Flickering transiently beneath the ways...”
+「まさしく、海のほたるじゃな」=“Truly, they are fireflies of the sea.”
+「そう……だね、大きさもふわふわ飛んでるあの蛍と同じくらいかな？」=“Yeah... I'd say they're about as big as those softly floating insects too?”
+「うむ、かもしれぬ」=“Umu, perhaps.”
+「この光の粒、ひとつひとつが命なのじゃな……」=“Each of these lights, is a life, isn't it...”
 「…………」=“...”
-//「見よ、懸命に瞬いておる。このほの暗い世界で己の存在を主張しようとするかのようじゃ」=
-//「俺はここにいるぞー、って？」=
-//「ふふ、私かもしれぬぞ」=
-//「しかし、実際、あやつらは何を思ってあのように光るのかのう？」=
-//「声が届くのであれば問いかけてみたいものじゃが」=
-//「もし理由を聞いたりしたら、きょとんとしちゃうかもね。そんなの知らないよって」=
-//「己が存在することと、あのように光を放つのは、イコールである……ということかの」=
-//「ふむ、それは人に『なぜ生きるのか』と問うに等しい。つまらぬ問いかけをするところだったのじゃ」=
-//「……美しい」=
-//「海ほたる達の輝きに向ける言葉など、きっと……それくらいで充分なのじゃ」=
-//「あはは。随分と大人びたこと言うよね？」=
-//「わしはいつも大人なのじゃぞ？　まぁ、この場の雰囲気には合ってなかったかもしれぬが……」=
-//「なら、ここからは初々しい恋人同士ってことで」=
-//「……うむ、海ほたる達が彩ってくれたこの夜にしばし浸るとするのじゃ」=
-//「キレイだね」=
-//「…………そうじゃな、本当に」=
+「見よ、懸命に瞬いておる。このほの暗い世界で己の存在を主張しようとするかのようじゃ」=“Look, they are twinkling so brightly, making their existence known to this dark and gloomy world.”
+「俺はここにいるぞー、って？」=“I'm (ore) right here, you know!?”
```

Comment:
Is `(ore)` here just a translation note?

Reply:
Yeah, looks like a leftover note.